hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,178 @@
+    "shootName": "c-35a9898",
+    "status": "UpdateDetected"
+  },
+  {
+    "runtimeId": "exxe4b14-7bc2-4947-931c-f8673793b02f",
+    "shootName": "",
+    "status": "Error",
+    "errorMessage": "Failed to fetch shoot: shoot was deleted or the runtimeID is incorrect"
+  }
+]
+```
+
+The runtime with the `a774bae2-ed8b-464e-85cc-ab8acd4461d5` cannot be automatically restored as there were several updates to the shoot. The runtime must be fixed manually, as there is a risk some updates performed by the user will be overwritten.
+
+## Configurable Parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description                                                                                                                                                                                                                                                                         | Default value          |
+|------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
+| **kcp-kubeconfig-path** | Path to the Kubeconfig file of KCP cluster.                                                                                                                                                                                                                                         | `/path/to/kcp/kubeconfig` |
+| **gardener-kubeconfig-path** | Path to the Kubeconfig file of Gardener cluster.                                                                                                                                                                                                                                    | `/path/to/gardener/kubeconfig` |
+| **gardener-project-name** | Name of the Gardener project.                                                                                                                                                                                                                                                       | `gardener-project-name` |
+| **output-path** | Path where generated report, and yamls will be saved. Directory has to exist.                                                                                                                                                                                                       | `/tmp/`                |
+| **dry-run** | Dry-run flag. Has to be set to **false**, otherwise migrator will not apply the CRs on the KCP cluster.                                                                                                                                                                             | `true`                 |
```

Comment:
```suggestion
| **dry-run** | Dry-run flag. Must be set to **false**, otherwise the migrator does not apply the CRs on the KCP cluster.                                                                                                                                                                             | `true`                 |
```